homeassistant/components/mqtt/sensor.py

```diff
@@ -343,12 +343,12 @@ def _value_is_expired(self, *_):
         self.async_write_ha_state()
 
     @property
-    def native_unit_of_measurement(self):
+    def native_unit_of_measurement(self) -> str | None:
```

Comment:
pylint was locally show this as an issue

Reply:
_Not a review._

> pylint was locally show this as an issue

That's a custom plugin which isn't enabled for CI. It's used to help with better type annotations for default methods. You can safely remove the annotations here if you want. CI should still pass. _Of course, you can leave them just as well._